extensions/src/config/external-tools.js

```diff
@@ -0,0 +1,58 @@
+
```

Comment:
This file should be called just `index.js` due we don't have any configuration file.